percent_box.js

```diff
@@ -113,10 +113,19 @@ class PercentBoxElement extends HTMLElement {
       this._pbox.update(data);
       toggleSVGContainerAndIndicator.call(this, false);
       broker.addEventListener('stats-stream-data', (evt) => {
-        const stats = evt.detail;
-        const val = stats[this.percentKey];
-        const total = stats[this.totalKey];
-        data = [ val, total - val ];
+        const stats = evt.detail.stats;
+        const { mappedReads, unmappedReads } = evt.detail.mapDataFromIndex
```

Comment:
Adding this here makes <iobio-percent-box> aware of the concept of mapped reads. We want to keep it generic.

I think the proper place to handle this is in the data broker. Basically we need to add 2 new properties to `stats-stream-data`:

`calculated_mapped_reads` and `calculated_total_reads`.

If the mapped reads data is available from the index, we should set them according to the values from the index. If not, set them to the values from the sampled data.